using_images/other_images/jenkins.adoc

```diff
@@ -169,51 +162,109 @@ To create a new Jenkins application using standard Jenkins authentication:
 ----
 $ oc new-app -e \
     JENKINS_PASSWORD=<password> \
-    openshift/jenkins-1-centos7
+    openshift/jenkins-2-centos7
 ----
 
 [[jenkins-environment-variables]]
 === Environment Variables
 
 The Jenkins server can be configured with the following environment variables:
 
-.Jenkins Environment Variables
-[cols="4a,6a",options="header"]
-|===
+* `JENKINS_PASSWORD` (default: `password`)
++
+The password for the `admin` user when using standard Jenkins authentication.
+Not applicable when using {product-title} OAuth authentication.
 
-|Variable name |Description
+* `OPENSHIFT_ENABLE_OAUTH` (default: `false`)
++
+Determines whether the OpenShift Login plug-in manages authentication when
+logging into Jenkins. To enable, set to `true`.
 
-|`*JENKINS_PASSWORD*`
-|The password for the `*admin*` user when using standard Jenkins authentication.
-Not applicable when using {product-title} OAuth authentication.
+* `CONTAINER_HEAP_PERCENT` (default: `0.5`, i.e. 50%)
++
+Specifies the percentage of the container memory limit allocated to the Jenkins
+JVM maximum heap size (-Xmx).
```

Comment:
Need to discuss how this interacts with JAVA_MAX_HEAP_PARAM, JENKINS_JAVA_OPTIONS, and JENKINS_JAVA_OVERRIDES.